Userland/Applications/TextEditor/MainWidget.cpp

```diff
@@ -370,6 +369,13 @@ Web::OutOfProcessWebView& MainWidget::ensure_web_view()
     return *m_page_view;
 }
 
+FileSystemAccessClient::Client& MainWidget::ensure_file_system_access_client()
```

Comment:
I'm going to move this into `FileSystemAccessClient::Client` because I think every client is going to want this functionality.